heudiconv/info.py

```diff
@@ -27,7 +27,7 @@
     "dcmstack>=0.8",
     "etelemetry",
     "filelock>=3.0.12",
-    "nibabel",
+    "nibabel>=5.3.1",
```

Comment:
remind me -- why we did not have that issue before? if that is just a buggy singular nibabel version - we better just exclude that one

Reply:
Forgot to note the version that was causing this issue before upgrading, but has been there for a while https://github.com/nipy/heudiconv/issues/455#issuecomment-633105835 and my environment wasn't that old.  Also we had https://github.com/nipy/nibabel/pull/1378 recently merge that solved failing tests.

Reply:
well, this is a fresh version released not even a month ago. Some distributions might not have it yet, and would cause troubles.   I would just keep in mind that a recent nibabel is needed for processing enhanced dicoms?

May be even get some "catch all" handler that if exception happens and nibabel is below this version -- recommend upgrading?